src/einsteinpy/symbolic/ricci.py

```diff
@@ -83,7 +92,41 @@ def from_metric(cls, metric):
 
         """
         rc = ChristoffelSymbols.from_metric(metric)
-        return cls.from_christoffels(rc)
+        return cls.from_christoffels(rc, parent_metric=None)
```

Comment:
can you change the variable name from `rc` to `ch`, as `ch` is better fitted for christoffel symbols.